zh/deploy-tidb-cluster.md

```diff
@@ -4,3 +4,320 @@ summary: 了解如何在 Kubernetes 环境中部署 TiDB 集群。
 ---
 
 # 在 Kubernetes 上部署 TiDB 集群
+
+本文介绍如何在 Kubernetes 环境中部署 TiDB 集群。
+
+## 前提条件
+
+<!-- TODO -->
```

Comment:
```suggestion
- TiDB Operator [部署](deploy-tidb-operator.md)完成。
```

Reply:
Ref: https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/deploy-on-general-kubernetes/